agent/commands/setup.go

```diff
@@ -158,6 +158,8 @@ func register(cfg *config.Config, l *logrus.Entry) {
 
 	fmt.Printf("Registered.\n")
 	cfg.ID = agentID
+	cfg.Server.Username = "api_key"
+	cfg.Server.Password = token
```

Comment:
let's check that the token is not empty in this response. it might happen when PMM Server version is older then PMM client